README.md

```diff
@@ -78,6 +78,17 @@ Follow these steps to run the OSS Review Toolkit from source code:
 
    * `./gradlew cli:run --args="requirements"`
 
+Alternatively, you can also run the OSS Review Toolkit by building its Docker image:
+
+1. Ensure you have recent versions of Git and Docker installed.
+
+2. Clone this repository with submodules by running `git clone --recurse-submodules`. If you have already cloned
+   non-recursively, you can initialize submodules afterwards by running `git submodule update --init --recursive`.
```

Comment:
I have not tested it but I would assume that recursive cloning is not required because the submodules are only used for testing.

Reply:
I was not sure about this, hoped @sschuberth could provide clarity

Reply:
> recursive cloning is not required because the submodules are only used for testing.

That's correct. As of 7f373253dfa5ec93ea358087c70a6acdd6dbe4c7 `.gitmodules` only refers to test assets.